runtime/interpreter/value_string.go

```diff
@@ -108,6 +100,62 @@ func stringFunctionFromCharacters(invocation Invocation) Value {
 	return NewUnmeteredStringValue(builder.String())
 }
 
+var StringTypeJoinDefaultSeparator = NewUnmeteredStringValue(",")
+
+func stringFunctionJoin(invocation Invocation) Value {
+	argument, ok := invocation.Arguments[0].(*ArrayValue)
```

Comment:
maybe rename this variable `argument` to a more intuitive one like `stringArray` or  even just `array`

Reply:
Done.